aana/integrations/external/yt_dlp.py

```diff
@@ -10,6 +11,42 @@
     DownloadException,
 )
 
+class VideoInfo(TypedDict):
```

Comment:
Can we reuse VideoMetadata model and add duration to it?

Reply:
I thought of that. 
It will work just for the video serving sites like youtube, so I keep it to accept a url, the user would handle it itslef. because most of the time it will be an empty output.